app/src/main/java/com/duckduckgo/app/email/EmailManager.kt

```diff
@@ -41,6 +43,7 @@ interface EmailManager {
 
     fun signOut()
     fun getEmailAddress(): String?
+    fun getUserData(): String?
```

Comment:
nit: Signature doesn't need to be nullable
```suggestion
    fun getUserData(): String
```